### PR TITLE
docs: Add OpenLLM

### PR DESCRIPTION
**Pull request by Sherlock113**
Hi Qwen2.5 team,

This is Sherlock from the BentoML community. We would like to add [OpenLLM](https://github.com/bentoml/OpenLLM) as a deployment tool for Qwen2.5 in the readme. Qwen2.5 was already added to OpenLLM. Users can easily run it locally or deploy it to the cloud with OpenLLM.

In addition to the readme, we also would like to add OpenLLM to the Deployment section of the [Qwen2.5 doc](https://qwen.readthedocs.io/en/latest/index.html).

@jklj077 @JustinLin610  Let me know if you have questions. Thanks!

### CODE REVIEW

**Comment by jklj077** [comment]
Hi, thanks for this PR!
Would you mind keeping a minimal part in the readme and moving most of them to the docs (similar to TGI/SkyPilot)?

**Comment by Sherlock113** [comment]
@jklj077 Hi, thank you for the review! I updated the pr. It now has:

1. Minimal OpenLLM usage demo in readme
2. More details in the Deployment section of the doc
3. A Chinese version in doc

Please take a look again. Thanks.

File: README.md [hunk 1]
```diff
@@ -254,6 +254,41 @@ for m in state.messages():
...
+### OpenLLM
+
+[OpenLLM](https://github.com/bentoml/OpenLLM) allows you to easily run Qwen2.5 as OpenAI-compatible APIs. You can start a model server using `openllm serve`. For example:
+
+```bash
+openllm serve qwen2.5:7b
+```
+
+The server is active at `http://localhost:3000/`, providing OpenAI-compatible APIs. You can create an OpenAI client to call its chat API:
+
```
Comment (jklj077) [inline README.md:266]: the code in the following seems a bit repetitive to me. I think it's okay to just state that an OpenAI API is availble at ...

File: docs/source/deployment/openllm.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+       Send an HTTP request to its ``/generate`` endpoint via CURL:
+
+       .. code-block:: bash
+
+            curl -X 'POST' \
+               'http://localhost:3000/api/generate' \
+               -H 'accept: text/event-stream' \
+               -H 'Content-Type: application/json' \
+               -d '{
+               "prompt": "Explain superconductors like I am five years old",
```
Comment (jklj077) [inline docs/source/deployment/openllm.rst:80]: we mostly use the same example across the repo. do you mind using the same one?

File: docs/source/deployment/openllm.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+.. tab-set::
+
+    .. tab-item:: CURL
+
+       Send an HTTP request to its ``/generate`` endpoint via CURL:
+
+       .. code-block:: bash
+
+            curl -X 'POST' \
+               'http://localhost:3000/api/generate' \
```
Comment (jklj077) [inline docs/source/deployment/openllm.rst:76]: this is the custom API, right? I think
line 60 ```By default, the server starts at ``http://localhost:3000/``, providing OpenAI-compatible APIs.``` should be modified to be clearer.

File: docs/source/deployment/openllm.rst [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+               -d '{
+               "prompt": "Explain superconductors like I am five years old",
+               "model": "Qwen/Qwen2.5-7B-Instruct",
+               "max_tokens": 2048,
+               "stop": null
+            }'
+
+    .. tab-item:: Python client
+
+       Call the OpenAI-compatible endpoints with any frameworks and tools that support the OpenAI API protocol. Here is an example:
```
Comment (jklj077) [inline docs/source/deployment/openllm.rst:88]: nit: any is not necessary here

File: README.md (hunk 1 above)
Comment (Sherlock113) [reply README.md]: Deleted. Users can just check out the details in doc.

File: docs/source/deployment/openllm.rst (hunk 3 above)
Comment (Sherlock113) [reply docs/source/deployment/openllm.rst:76]: I see. Let me delete `providing OpenAI-compatible APIs`. Readers can just see the examples to call APIs.

**Comment by Sherlock113** [comment]
@jklj077 I updated the pr based on your comments. Thanks.

**Pull request merged**